mill/__init__.py

```diff
@@ -530,6 +530,48 @@ async def set_heater_temp(self, device_id: str, set_temp: float) -> None:
             self.devices[device_id].is_heating = set_temp > self.devices[device_id].current_temp
             self.devices[device_id].last_fetched = dt.datetime.now(dt.timezone.utc)
 
+    async def _patch_device_settings(self, device_id: str, settings: dict[str, Any]) -> bool:
+        """PATCH /devices/{id}/settings for heaters and sockets."""
+        device = self.devices.get(device_id)
+        if not device or not isinstance(device, (Heater, Socket)):
+            _LOGGER.error("Device id %s not found or unsupported", device_id)
+            return False
+
+        enabled = bool(getattr(device, "power_status", True))
+
+        payload: dict[str, Any] = {
+            "deviceType": device.deviceType,
+            "enabled": enabled,
+            "settings": settings,
+        }
+
+        resp = await self.request(f"devices/{device_id}/settings", payload, patch=True)
+        if resp is None:
+            _LOGGER.error("Failed to patch settings for %s. Payload=%s", device_id, settings)
+            return False
+
+        if isinstance(resp, dict) and resp.get("success") is False:
```

Comment:
Does Mill API actually return {"success": false} responses? 

I cant see that in the documentation?